#!/usr/bin/python3
# 6-raise_exception_msg.py


def raise_exception_msg(message=""):
    """Raising a NameError exception with custom message"""
    raise NameError(message)
